pkg/keystore/test/test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/ethersphere/bee/pkg/keystore"
 )
 
+// Service is utility testing function that can be used to test implementations
+// of keystore.Service interface.
```

Comment:
// Service is a utility testing function that can be used to test implementations
// of the keystore.Service interface.